sdk/loadtestservice/test-resources.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Please delete this file for now, in management plane we run the tests in playback mode and don't need this file.